packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -82,9 +82,6 @@ export class ApiBuilder {
      */
     this.app.use((req: Request, res: Response, next: NextFunction) => {
       if (options.attachModels) {
-        winston.warn(
-          "Best practices say orchestrator servers shouldn't access the db directly, are you sure you need req.models?",
-        );
```

Comment:
I thought this would only run on start up but it actually runs every request which is _way_ too spammy